File: migration/pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    //type
    float c;
    // get change
    do
    {
        c = get_float("Change owed: ");
    }
    while (c <= 0);
    c = round(c * 100);
    int q = c / 25;
    int d = (c - (q * 25)) / 10;
    int n = (c - (q * 25) - (d * 10)) / 5;
    int p = c - (q * 25) - (d * 10) - (n * 5);
    // how many coins you have to give
    printf("%i\n", q + d + n + p);
}

File: migration/pset6/cash/cash.py
from cs50 import get_float

# main


def main():
    n = get_change()
    n = round(n*100)
    q = int(n / 25)
    d = int((n - (q * 25)) / 10)
    i = int((n - (q * 25) - (d * 10)) / 5)
    p = int(n - (q * 25) - (d * 10) - (i * 5))
    print(f"{q + d + i + p}")

# get input


def get_change():
    while True:
        n = get_float("Change owed: ")
        if n > 0:
            break
    return n


# execute
main()